test/unit/org/apache/cassandra/repair/RepairJobTest.java

```diff
@@ -152,6 +182,7 @@ public void setup()
     @After
     public void reset()
     {
+        session.onSyncComplete.clear();
```

Comment:
nit: Not entirely necessary, given we create a new session every test w/ `@Before`?

Reply:
removed